roles/openshift_version/tasks/main.yml

```diff
@@ -142,7 +142,7 @@
         msg: "openshift_image_tag was not defined. Falling back to v{{ openshift_version }}"
 
     - set_fact:
-        openshift_image_tag: v{{ openshift_version }}
+        openshift_image_tag: v{{ ( openshift_version ).split('-') }}
```

Comment:
That would turn `openshift_version` into an array.

Also, message above is not changed